packages/appstore/src/components/onboarding-new/static-dashboard.tsx

```diff
@@ -66,22 +66,22 @@ const StaticDashboard = ({
     const { client, traders_hub } = useStore();
     const { content_flag, CFDs_restricted_countries, financial_restricted_countries } = traders_hub;
     const { is_eu_country, is_logged_in } = client;
-
-    const [index, setIndex] = React.useState<number>(0);
-
-    const Divider = () => <div className='divider' />;
-
     const eu_user =
         content_flag === ContentFlag.LOW_RISK_CR_EU ||
         content_flag === ContentFlag.EU_REAL ||
         content_flag === ContentFlag.EU_DEMO;
-
     const is_eu_user = (is_logged_in && eu_user) || (!is_logged_in && is_eu_country);
 
-    const toggle_options = [
-        { text: `${is_eu_user ? 'Multipliers' : 'Options & Multipliers'}`, value: 0 },
-        { text: 'CFDs', value: 1 },
+    const [index, setIndex] = React.useState<number>(is_eu_user ? 1 : 0);
+
+    const Divider = () => <div className='divider' />;
+
+    const getToggleOptions = () => [
+        { text: is_eu_user ? localize('Multipliers') : localize('Options & Multipliers'), value: 0 },
+        { text: localize('CFDs'), value: 1 },
```

Comment:
can we check the value here?
The type of  value in Buttonsarray  is declared as string in ButtonToggle component

Reply:
made type changes in ButtonToggle component (since this component already had type of value set to `value: number | string;`).